File: app/models/dummy/Dissemination.java
package models.dummy;

import models.IArchive;
import models.IDissemination;

import java.util.Date;

/**
 * Created by Beemen on 06/08/2015.
 */
public class Dissemination implements IDissemination{

    Date _CreatedDate;
    @Override
    public Date CreatedDate() {
        return _CreatedDate;
    }

    @Override
    public void CreatedDate(Date value) {
        _CreatedDate = value;
    }

    public String _DummyDescription;
    @Override
    public String ToString() {
        return _DummyDescription;
    }
}

File: Models/dummy/Dissemination.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace end_user_gui.Models.dummy
{
    public class Dissemination : IDissemination
    {
        public Guid UUID = Guid.NewGuid();

        public DateTime CreatedDate { get; set; }

        public String _DummyDescription;

        public override String ToString()
        {
            return _DummyDescription;
        }

        public String KeyString
        {
            get
            {
                return UUID.ToString();
            }
        }
    }
}